app/models/cloud_volume.rb

```diff
@@ -25,6 +25,13 @@ class CloudVolume < ApplicationRecord
 
   delegate :queue_name_for_ems_operations, :to => :ext_management_system, :allow_nil => true
 
+  supports_not :safe_delete
+  virtual_column :supports_safe_delete,  :type => :boolean
```

Comment:
`,  :type =>`

Looks like some extra spaces here, are you aligning with something outside this diff?